src/app/modules/games/modules/add-game/add-game-store.js

```diff
@@ -13,14 +13,10 @@ export default {
   },
   actions: {
     [AddGameStoreKeys.Actions.SaveGame]: function({ commit, dispatch }, game) {
-      addGameService.createGame(
-        game,
-        () => {
-          Router.replace('/games/listing');
-          dispatch(GamesListingStoreKeys.Actions.RetrieveGames);
-        },
-        error => commit(AddGameStoreKeys.Mutations.SaveGameFailure, error)
-      );
+      return addGameService.createGame(game)
```

Comment:
[vuex actions can return promises](https://vuex.vuejs.org/guide/actions.html#composing-actions) - I think this can help clean up the control flow a little better, and also make it easier to compose an action chain if / when needed